### PR TITLE
feat(frontend): Set up Btc Transactions page

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

The transactions page for Bitcoin was not showing balance, nor nice background.

The error was that it was using the ETH transactions page and that the hero didn't have an assigned background for btc.

# Changes

* Create new component BtcTransactions.
* Use new component in TransactionsSignedIn.
* Add a background when the hero is for "btc".

# Tests

Tested locally.

### CODE REVIEW

**Comment by lmuntaner** [comment]
UI:

Screenshot 2024-09-04 at 09 11 38

**Comment by lmuntaner** [comment]
New UI:

Screenshot 2024-09-04 at 11 06 51

File: src/frontend/src/btc/components/BtcTransactions.svelte [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<script lang="ts">
+	import Header from '$lib/components/ui/Header.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+</script>
+
+<Header>
+	<h2 class="text-base">{$i18n.transactions.text.title}</h2>
```
Comment (peterpeterparker) [inline src/frontend/src/btc/components/BtcTransactions.svelte:7]: That's actually an error in `IcTransactions` page which I guess you used as inspiration to create this page. It should be just the text - no `h2`.

Can you modify it and also provide a fix for the other page if you do not mind?

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

Minor comment to resolve before merge.

File: src/frontend/src/btc/components/BtcTransactions.svelte (hunk 1 above)
Comment (lmuntaner) [reply src/frontend/src/btc/components/BtcTransactions.svelte]: Done!

**Pull request merged**